site/src/examples/parent-child-layout/PreferencesDialog.tsx

```diff
@@ -154,22 +157,23 @@ export const PreferencesLayout = () => {
     },
   ];
 
-  const [currentView, setCurrentView] = useState<
-    StackedViewElement | undefined
-  >();
+  const [collapsedView, setcollapsedView] = useState<"child" | "parent">(
```

Comment:
```suggestion
  const [collapsedView, setCollapsedView] = useState<"child" | "parent">(
```
This should be camel case